src/Is/fqdn.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * @since 0.0.2
+ */
+
+import { isString } from "./string";
+
+/**
+ * check if it's Fully qualified domain name
+ *
+ * @since 0.0.2
+ * @category Is
+ */
+export const isFqdn = <A>(val: A) => {
+  if (!val || !isString(val)) {
+    return false;
+  }
+  const _val: string = "" + val;
```

Comment:
is it required since we already handle the type validation on line 14?

Reply:
ts doesn't aware that val should be considered as string, add empty string with any value will convert any value into string.